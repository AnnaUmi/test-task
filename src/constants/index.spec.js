import { noop } from './index';

describe('constants', () => {
  it('noop should not change', () => {
    expect(noop()).toMatchSnapshot();
  });
});
