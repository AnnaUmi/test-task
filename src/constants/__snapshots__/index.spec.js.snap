// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constants noop should not change 1`] = `undefined`;
